packages/contract-schema/index.js

```diff
@@ -176,7 +176,8 @@ var properties = {
   },
   networkType: {},
   devdoc: {},
-  userdoc: {}
+  userdoc: {},
+  "db:contract": {}
```

Comment:
I've considered this a bunch... we don't use the term "DB" for anything else right now. @truffle/db doesn't even call anything internally a DB. I'm happy if this never changes; I don't see why Truffle would be forced to call something a DB alone, rather than naming such a component in relation to its purpose (e.g., if there becomes some external persistence system for some other unrelated thing, then just calling that thing a "DB" is probably insufficient).

Ultimately, my perspective is that it benefits us to align identifiers with the brand name, even if it's not technically correct.

Reply:
Basically, to Truffle, @truffle/db is the database. It's not just an ORM because an ORM carries the notion of generality. Instead: there's one data model, and it's Truffle's data model. @truffle/db is what stores that data model for Truffle.

Reply:
Fair arguments. Any response I have to this is really just down to being pedantic. I'm fine with what you have. It's just confusing for people that don't know the architecture of truffle that `db` is Truffle DB rather than some generic reference to another database. It's just ambiguous because you're using the ambiguous term, not the branded term

Reply:
I hear that. My hope is that it's no worse than our other generic terms that we use everywhere that you just kind of have to realize "this is a truffle thing"